### DISCUSSION TITLE
Where do I go from here?

### DISCUSSION THREAD

**Discussion by Danogbans** [EddieHubCommunity/support#7885] [First-Timers] [answered]
Hi everyone,
Hope you are all doing great. I am new in the community and I have just finished the hactoberfest-practice. So, what do you suggest should be the next step to take?
Thanks.

**Comment by Pradumnasaraf** [comment, marked as answer]
Hey, 
You will find further steps on the community GitHub page (https://github.com/EddieHubCommunity?view_as=public).

<img width="856" alt="Screenshot 2023-07-21 at 9 49 02 PM" src="https://github.com/EddieHubCommunity/support/assets/51878265/e098f59f-be0e-42a0-9071-22bb970ee8c9">

**Comment by Danogbans** [comment]
Thanks Pradumnasaraf!
